src/hotspot/share/gc/shared/genCollectedHeap.cpp

```diff
@@ -678,6 +679,8 @@ void GenCollectedHeap::do_collection(bool           full,
 
     print_heap_after_gc();
   }
+  // update the live size after last GC
+  _live_size = _young_gen->live() + _old_gen->live();
```

Comment:
I would prefer if that code were placed into `gc_epilogue`.

Reply:
👍